src/sentry/static/sentry/app/components/debugFileFeature.tsx

```diff
@@ -31,9 +31,17 @@ const DebugFileFeature = ({available, feature}: Props) => {
   let icon: React.ReactNode = null;
 
   if (available === true) {
-    icon = <Icon type="success" src="icon-checkmark-sm" />;
+    icon = (
+      <IconWrapper>
+        <IconCheckmark size="sm" color="green500" />
```

Comment:
I believe that `size="sm"` is the default, so theoretically we don't have to pass that. (I like how explicit it is though).